Oct 29 15:33:28 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:33:31 v-guard0001 python3[815]:  * Serving Flask app 'main'
Oct 29 15:33:31 v-guard0001 python3[815]:  * Debug mode: on
Oct 29 15:33:32 v-guard0001 python3[824]: /home/vgadmin/vision-guard/main.py:26: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:33:32 v-guard0001 python3[824]:   GPIO.setup(config.PIN_BUZZ, GPIO.OUT)
Oct 29 15:33:32 v-guard0001 python3[824]: /home/vgadmin/vision-guard/main.py:28: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:33:32 v-guard0001 python3[824]:   GPIO.setup(config.PIN_R_LED, GPIO.OUT)
Oct 29 15:33:32 v-guard0001 python3[824]: /home/vgadmin/vision-guard/main.py:29: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:33:32 v-guard0001 python3[824]:   GPIO.setup(config.PIN_Y_LED, GPIO.OUT)
Oct 29 15:33:32 v-guard0001 python3[824]: /home/vgadmin/vision-guard/main.py:30: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:33:32 v-guard0001 python3[824]:   GPIO.setup(config.PIN_G_LED, GPIO.OUT)
Oct 29 15:50:16 v-guard0001 python3[1411]: Traceback (most recent call last):
Oct 29 15:50:16 v-guard0001 python3[1411]:   File "/home/vgadmin/vision-guard/main.py", line 5, in <module>
Oct 29 15:50:16 v-guard0001 python3[1411]:     import config
Oct 29 15:50:16 v-guard0001 python3[1411]:   File "/home/vgadmin/vision-guard/config.py", line 26, in <module>
Oct 29 15:50:16 v-guard0001 python3[1411]:     STATUS_CRITICAL
Oct 29 15:50:16 v-guard0001 python3[1411]: NameError: name 'STATUS_CRITICAL' is not defined
Oct 29 15:50:16 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:50:16 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:50:16 v-guard0001 systemd[1]: vg.service: Consumed 20.827s CPU time.
Oct 29 15:50:22 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 1.
Oct 29 15:50:22 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:50:22 v-guard0001 systemd[1]: vg.service: Consumed 20.827s CPU time.
Oct 29 15:50:22 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:50:24 v-guard0001 python3[1420]: /home/vgadmin/vision-guard/main.py:26: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:24 v-guard0001 python3[1420]:   GPIO.setup(config.PIN_BUZZ, GPIO.OUT)
Oct 29 15:50:24 v-guard0001 python3[1420]: /home/vgadmin/vision-guard/main.py:28: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:24 v-guard0001 python3[1420]:   GPIO.setup(config.PIN_R_LED, GPIO.OUT)
Oct 29 15:50:24 v-guard0001 python3[1420]: /home/vgadmin/vision-guard/main.py:29: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:24 v-guard0001 python3[1420]:   GPIO.setup(config.PIN_Y_LED, GPIO.OUT)
Oct 29 15:50:24 v-guard0001 python3[1420]: /home/vgadmin/vision-guard/main.py:30: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:24 v-guard0001 python3[1420]:   GPIO.setup(config.PIN_G_LED, GPIO.OUT)
Oct 29 15:50:24 v-guard0001 python3[1420]:  * Serving Flask app 'main'
Oct 29 15:50:24 v-guard0001 python3[1420]:  * Debug mode: on
Oct 29 15:50:26 v-guard0001 python3[1436]: /home/vgadmin/vision-guard/main.py:26: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:26 v-guard0001 python3[1436]:   GPIO.setup(config.PIN_BUZZ, GPIO.OUT)
Oct 29 15:50:26 v-guard0001 python3[1436]: /home/vgadmin/vision-guard/main.py:28: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:26 v-guard0001 python3[1436]:   GPIO.setup(config.PIN_R_LED, GPIO.OUT)
Oct 29 15:50:26 v-guard0001 python3[1436]: /home/vgadmin/vision-guard/main.py:29: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:26 v-guard0001 python3[1436]:   GPIO.setup(config.PIN_Y_LED, GPIO.OUT)
Oct 29 15:50:26 v-guard0001 python3[1436]: /home/vgadmin/vision-guard/main.py:30: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:26 v-guard0001 python3[1436]:   GPIO.setup(config.PIN_G_LED, GPIO.OUT)
Oct 29 15:50:32 v-guard0001 python3[1481]: /home/vgadmin/vision-guard/main.py:26: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:32 v-guard0001 python3[1481]:   GPIO.setup(config.PIN_BUZZ, GPIO.OUT)
Oct 29 15:50:32 v-guard0001 python3[1481]: /home/vgadmin/vision-guard/main.py:28: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:32 v-guard0001 python3[1481]:   GPIO.setup(config.PIN_R_LED, GPIO.OUT)
Oct 29 15:50:32 v-guard0001 python3[1481]: /home/vgadmin/vision-guard/main.py:29: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:32 v-guard0001 python3[1481]:   GPIO.setup(config.PIN_Y_LED, GPIO.OUT)
Oct 29 15:50:32 v-guard0001 python3[1481]: /home/vgadmin/vision-guard/main.py:30: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Oct 29 15:50:32 v-guard0001 python3[1481]:   GPIO.setup(config.PIN_G_LED, GPIO.OUT)
Oct 29 15:50:46 v-guard0001 python3[1523]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:50:46 v-guard0001 python3[1523]:     elif config.ge
Oct 29 15:50:46 v-guard0001 python3[1523]:                   ^
Oct 29 15:50:46 v-guard0001 python3[1523]: SyntaxError: expected ':'
Oct 29 15:50:47 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:50:47 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:50:47 v-guard0001 systemd[1]: vg.service: Consumed 14.994s CPU time.
Oct 29 15:50:52 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 2.
Oct 29 15:50:52 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:50:52 v-guard0001 systemd[1]: vg.service: Consumed 14.994s CPU time.
Oct 29 15:50:52 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:50:52 v-guard0001 python3[1530]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:50:52 v-guard0001 python3[1530]:     elif config.ge
Oct 29 15:50:52 v-guard0001 python3[1530]:                   ^
Oct 29 15:50:52 v-guard0001 python3[1530]: SyntaxError: expected ':'
Oct 29 15:50:52 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:50:52 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:50:57 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 3.
Oct 29 15:50:57 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:50:57 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:50:57 v-guard0001 python3[1539]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:50:57 v-guard0001 python3[1539]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:50:57 v-guard0001 python3[1539]:                                                       ^
Oct 29 15:50:57 v-guard0001 python3[1539]: SyntaxError: expected ':'
Oct 29 15:50:57 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:50:57 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:02 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 4.
Oct 29 15:51:02 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:02 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:02 v-guard0001 python3[1542]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:02 v-guard0001 python3[1542]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:02 v-guard0001 python3[1542]:                                                       ^
Oct 29 15:51:02 v-guard0001 python3[1542]: SyntaxError: expected ':'
Oct 29 15:51:02 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:02 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:07 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 5.
Oct 29 15:51:07 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:07 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:08 v-guard0001 python3[1545]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:08 v-guard0001 python3[1545]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:08 v-guard0001 python3[1545]:                                                       ^
Oct 29 15:51:08 v-guard0001 python3[1545]: SyntaxError: expected ':'
Oct 29 15:51:08 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:08 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:13 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 6.
Oct 29 15:51:13 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:13 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:13 v-guard0001 python3[1549]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:13 v-guard0001 python3[1549]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:13 v-guard0001 python3[1549]:                                                       ^
Oct 29 15:51:13 v-guard0001 python3[1549]: SyntaxError: expected ':'
Oct 29 15:51:13 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:13 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:18 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 7.
Oct 29 15:51:18 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:18 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:18 v-guard0001 python3[1552]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:18 v-guard0001 python3[1552]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:18 v-guard0001 python3[1552]:                                                       ^
Oct 29 15:51:18 v-guard0001 python3[1552]: SyntaxError: expected ':'
Oct 29 15:51:18 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:18 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:23 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 8.
Oct 29 15:51:23 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:23 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:23 v-guard0001 python3[1556]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:23 v-guard0001 python3[1556]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:23 v-guard0001 python3[1556]:                                                       ^
Oct 29 15:51:23 v-guard0001 python3[1556]: SyntaxError: expected ':'
Oct 29 15:51:23 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:23 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:28 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 9.
Oct 29 15:51:28 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:28 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:29 v-guard0001 python3[1559]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:29 v-guard0001 python3[1559]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:29 v-guard0001 python3[1559]:                                                       ^
Oct 29 15:51:29 v-guard0001 python3[1559]: SyntaxError: expected ':'
Oct 29 15:51:29 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:29 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:34 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 10.
Oct 29 15:51:34 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:34 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:34 v-guard0001 python3[1562]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:34 v-guard0001 python3[1562]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:34 v-guard0001 python3[1562]:                                                       ^
Oct 29 15:51:34 v-guard0001 python3[1562]: SyntaxError: expected ':'
Oct 29 15:51:34 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:34 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:39 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 11.
Oct 29 15:51:39 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:39 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:39 v-guard0001 python3[1565]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:39 v-guard0001 python3[1565]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:39 v-guard0001 python3[1565]:                                                       ^
Oct 29 15:51:39 v-guard0001 python3[1565]: SyntaxError: expected ':'
Oct 29 15:51:39 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:39 v-guard0001 systemd[1]: [0;1;38;5;185m[0;1;39m[0;1;38;5;185mvg.service: Failed with result 'exit-code'.[0m
Oct 29 15:51:44 v-guard0001 systemd[1]: vg.service: Scheduled restart job, restart counter is at 12.
Oct 29 15:51:44 v-guard0001 systemd[1]: Stopped vg.service - Vision Guard Client.
Oct 29 15:51:44 v-guard0001 systemd[1]: Started vg.service - Vision Guard Client.
Oct 29 15:51:44 v-guard0001 python3[1568]:   File "/home/vgadmin/vision-guard/main.py", line 104
Oct 29 15:51:44 v-guard0001 python3[1568]:     elif config.get_status() == config.STATUS_CRITICAL
Oct 29 15:51:44 v-guard0001 python3[1568]:                                                       ^
Oct 29 15:51:44 v-guard0001 python3[1568]: SyntaxError: expected ':'
Oct 29 15:51:44 v-guard0001 systemd[1]: [0;1;39m[0;1;31m[0;1;39mvg.service: Main process exited, code=exited, status=1/FAILURE[0m
Oct 29 15:51:44 v-guard0001 systemd[1]: [0;1;